### DISCUSSION TITLE
Data type using in Plot

### DISCUSSION THREAD

**Discussion by greedyhao** [emilk/egui#626] [Q&A]
I find that the data format used by the Plot is floating point, but my data is integer. What should I do

**Comment by charleschege** [comment]
I have not used plot, but you could try to convert your integer to float like
```rs
let my_integer = 0_i32;
let my_float = my_integer as f32;
```

**Reply by greedyhao** [reply]
But there is another problem. The curve is not smooth.

![image](https://user-images.githubusercontent.com/25771942/129703878-093435d4-3d0a-494a-a916-cd2d1b794c8d.png)

**Reply by charleschege** [reply]
@emilk might have a better idea about that
